### DISCUSSION TITLE
Option to debug

### DISCUSSION THREAD

**Discussion by BeckeBauer** [globaleaks/globaleaks-whistleblowing-software#4189] [Q&A] [answered]
I have created a questionnaire and reporting works fine. However, when the recipient wants to process the report, e.g. change status, export, edit expiration date, nothing happens and the page freezes. Also, at the end of the page there is no field to communicate with the reporting person. 

If I use the standard questionnaire GLOBALEAKS, everything works as expected (status can be changes, a message can be left to the reporting person etc.).

It seems that I somehow misconfigured my own questionnaire but I have no clue how to debug. Any help would be appreciated.

**Comment by evilaliv3** [comment]
@BeckeBauer: Which version of the software are you using?

**Reply by BeckeBauer** [reply]
Thank you for the quick feedback. I am on v5.0.3

But I think I found the problem: I emptied the name field and just added (a lot of) text to the description field since I wanted to have a conclusory remark at the end. However, it seems that questions must not have an empty name field and since I have to pick a question type, an option as well. 

Is there a way to just show some text at the end of the questionnaire (without the need to define any option)?

**Reply by evilaliv3** [reply, marked as answer]
No actually there is currently no feature that allows that
